Mathlib/Algebra/Category/ModuleCat/Presheaf.lean

```diff
@@ -376,52 +388,23 @@ noncomputable def toCorePresheafOfModules : CorePresheafOfModules R where
   obj X := (M.obj X).carrier
   map {X Y} f := (ModuleCat.semilinearMapAddEquiv (R.map f) (M.obj X) (M.obj Y)).symm (M.map f)
 
-end BundledCorePresheafOfModules
-
-namespace PresheafOfModules
-
 /-- Constructor for `PresheafOfModules R` based on a collection of objects
 of type `ModuleCat (R.obj X)` for all `X`, and restriction maps expressed
 as linear maps to restriction of scalars, see
 the structure `BundledCorePresheafOfModules`. -/
-noncomputable def mk'' (M : BundledCorePresheafOfModules R) : PresheafOfModules R :=
-  mk' M.toCorePresheafOfModules
+noncomputable def toPresheafOfModules : PresheafOfModules R :=
+  M.toCorePresheafOfModules.toPresheafOfModules
 
 @[simp]
-lemma mk''_obj (M : BundledCorePresheafOfModules R) (X : Cᵒᵖ) :
-    (mk'' M).obj X = (M.obj X).carrier := rfl
+lemma toPresheafOfModules_obj (X : Cᵒᵖ) :
+    M.toPresheafOfModules.obj X = (M.obj X).carrier := rfl
 
 @[simp]
-lemma mk''_presheaf_map_apply (M : BundledCorePresheafOfModules R)
-    {X Y : Cᵒᵖ} (f : X ⟶ Y) (x : M.obj X) :
-    (mk'' M).presheaf.map f x = M.map f x := rfl
+lemma toPresheafOfModules_presheaf_map_apply {X Y : Cᵒᵖ} (f : X ⟶ Y) (x : M.obj X) :
+    M.toPresheafOfModules.presheaf.map f x = M.map f x := rfl
 
 @[simp]
-lemma restrictionApp_mk'' (M : BundledCorePresheafOfModules R) {X Y : Cᵒᵖ} (f : X ⟶ Y) :
-    restrictionApp f (mk'' M) = M.map f := rfl
-
-namespace Hom
+lemma restrictionApp_toPresheafOfModules (M : BundledCorePresheafOfModules R) {X Y : Cᵒᵖ} (f : X ⟶ Y) :
+    PresheafOfModules.restrictionApp f M.toPresheafOfModules = M.map f := rfl
 
-variable {P Q : PresheafOfModules R}
-  (app : ∀ X, P.obj X →ₗ[R.obj X] Q.obj X)
-  (naturality : ∀ ⦃X Y : Cᵒᵖ⦄ (f : X ⟶ Y),
-    restrictionApp f P ≫ (ModuleCat.restrictScalars (R.map f)).map (app Y) =
-      ModuleCat.ofHom (app X) ≫ restrictionApp f Q)
-
-/-- A constructor for morphisms in `PresheafOfModules R` that is based on the data
-of a family of linear maps over the various rings `R.obj X`, and for which the
-naturality condition is stated using the restriction of scalars. -/
-def mk'' : P ⟶ Q where
-  hom :=
-    { app := fun X => (app X).toAddMonoidHom
-      naturality := fun X Y f => by
-        ext x
-        exact congr_hom (naturality f) x }
-  map_smul X := (app X).map_smul
-
-@[simp]
-lemma mk''_app : (mk'' app naturality).app = app := rfl
```

Comment:
Just making sure: You made `mk''` an `abbrev` so this is no longer needed?

Reply:
Yes!